zhaquirks/tuya/ts011f_plug.py

```diff
@@ -95,6 +95,73 @@ class Plug(EnchantedDevice):
     }
 
 
+class CircuitBreaker(EnchantedDevice):
+    """Tuya smart circuit breaker with metering (_TZ3000_qeuvnohg) https://www.aliexpress.com/item/1005004747066832.html."""
```

Comment:
I wonder, is it a permanent link or it can become inaccessible earlier than the product itself?

Reply:
I would remove it and also the `_TZ3000_qeuvnohg` reference.

Reply:
@zleba thanks for your contribution, maybe point to [Tongou TO-Q-SY1-JZT](https://zigbee.blakadder.com/Tongou_TO-Q-SY1-JZT.html) rather?